pkg/virtctl/vmexport/vmexport.go

```diff
@@ -725,7 +762,7 @@ func GetManifestUrlsFromVirtualMachineExport(vmexport *exportv1.VirtualMachineEx
 
 // waitForVirtualMachineExport waits for the VirtualMachineExport status and external links to be ready
 func waitForVirtualMachineExport(client kubecli.KubevirtClient, vmeInfo *VMExportInfo, interval, timeout time.Duration) error {
-	err := wait.PollImmediate(interval, timeout, func() (bool, error) {
+	err := wait.PollUntilContextTimeout(context.Background(), interval, timeout, false, func(ctx context.Context) (bool, error) {
```

Comment:
Previously this started polling immeditately, now it does not (4th arg set to `false`)